lib/theme_check/shopify_liquid/source_manager.rb

```diff
@@ -1,81 +1,110 @@
 # frozen_string_literal: true
 
+require 'net/http'
 require 'pathname'
+require 'tmpdir'
 
 module ThemeCheck
   module ShopifyLiquid
-    module SourceManager
-      extend self
-
-      def download_or_refresh_files
-        if has_required_files?
-          # TODO: https://github.com/Shopify/theme-check/issues/651
-          #
-          # Refresh files if they exist locally
-        else
-          download
+    class SourceManager
+      REQUIRED_FILE_NAMES = %i(filters objects tags latest).freeze
```

Comment:
/nit – I have "Format on save" activated and it transforms me this line into
`REQUIRED_FILE_NAMES = [:filters, :objects, :tags, :latest].freeze`
Not sure where this rule is defined though, Rubocop doesn't complain :)